types/components/messages.d.ts

```diff
@@ -4,6 +4,15 @@ export interface PayPalMessagesComponentOptions {
     style?: {
         layout?: "text" | "flex" | "custom";
         color?: string;
+        logo?: {
+            type?: 'primary' | 'alternative' | 'inline' | 'none';
+            position?: 'left' | 'right' | 'top';
+        };
+        text?: {
+            color?: 'black' | 'white' | 'monochrome' | 'grayscale';
+            size?: 	10 | 11 | 12 | 13 | 14 | 15 | 16;
```

Comment:
Nice work with the definition of the type. Minor detail, format the file to avoid extra tabs in `text.size` definition.

Reply:
Okay, I've commit [1d2c453](https://github.com/paypal/paypal-js/pull/167/commits/1d2c453a94ec391932a3af3056b1c1615e2f435a) to format.